va/va.go

```diff
@@ -87,13 +87,17 @@ func certNames(cert *x509.Certificate) string {
 		names = append(names, cert.Subject.CommonName)
 	}
 	names = append(names, cert.DNSNames...)
+	for i := range cert.IPAddresses {
+		names = append(names, cert.IPAddresses[i].String())
+	}
 	return strings.Join(names, ", ")
 }
 
 type vaTask struct {
-	Identifier string
-	Challenge  *core.Challenge
-	Account    *core.Account
+	Identifier     string
+	IdentifierType string
```

Comment:
I think it makes sense to add a new type to the `acme` package called `Identifier` that holds both a `Value` (called`Identifier` here) and a `Type` (called `IdentifierType` here) field. The `vaTask` can be updated to have a single `Identifier` field of type `acme.Identifier`.